lib/cogctl/actions/relay_groups/util.ex

```diff
@@ -12,29 +12,27 @@ defmodule Cogctl.Actions.RelayGroups.Util do
     Table.format(group_info, sort) |> display_output
   end
 
-  def render({relay_messages, error_count}, group_info, sort) do
-    relay_message = Enum.join(relay_messages, "\n")
-    Table.format(group_info, sort) <> "\n\n" <> relay_message
-    |> display_output
-    if error_count > 0 do
-      :error
-    else
-      :ok
-    end
+  # If the errorcount is not equal to 0, be sure that an error code
+  # is returned upon execution
+  def render({relay_messages, 0}, group_info, sort) do
+    render_output(relay_messages, group_info, sort)
+  end
+  def render({relay_messages, _errorcount}, group_info, sort) do
+    render_output(relay_messages, group_info, sort)
```

Comment:
No, they don't 😞. I can change that when I do the rendering output PR. I'll make a note.